adoc/chapters/programming_interface.adoc

```diff
@@ -6954,6 +6954,20 @@ If this is a <<ranged-accessor>>, returns the offset that was specified when the
 accessor was constructed.  For other accessors, returns the default constructed
 [code]#id<Dimensions>{}#.
 
+a@
+[source]
+----
+accessor_ptr<access::decorated::legacy> get_pointer() const noexcept
+----
+   a@ Returns a [code]#multi_ptr# to the start of this accessor's underlying
+      buffer, even if this is a <<ranged-accessor>> whose range does not start
+      at the beginning of the buffer.  The return value is unspecified if the
+      accessor is empty.
+
+This function may only be called from within a <<command>>.
+
+Deprecated in SYCL 2020.  Use [code]#get_multi_ptr# instead.
+
```

Comment:
I think it makes sense to remove `get_pointer` from Table 80 "Common buffer and local accessor member functions" (section 4.7.6.12).

You have already added a description here that is specific to `accessor`.  You should also add descriptions to:

* Table 72 "Member functions of the `host_accessor` class" (section 4.7.6.10.1)
* Table 78 "Member functions of the `local_accessor` class" (section 4.7.6.11.1)

Reply:
That's a good point, `get_pointer` is no longer common, I've made this change in https://github.com/KhronosGroup/SYCL-Docs/pull/432/commits/b84963dbb7a353890317fcaa01c8c172a727ae68.